src/app/public/modules/progress-indicator/progress-indicator-item/progress-indicator-item.component.html

```diff
@@ -1,24 +1,28 @@
-<div *ngIf="!isHorizontal || isActive"
-  class="sky-progress-indicator-item-heading"
-  [ngClass]="headingClassList"
+<div *ngIf="isVisible"
+  class="sky-progress-indicator-item"
+  [ngClass]="'sky-progress-indicator-item-status-' + statusName"
 >
-  <div
-    class="sky-progress-indicator-item-title"
-    [ngClass]="{
-      'sky-emphasized': isHorizontal,
-      'sky-deemphasized': isPassive && isActive
-    }"
+  <sky-progress-indicator-status-marker *ngIf="showStatusMarker"
+    [status]="status"
   >
-    {{ displayTitle }}
-  </div>
-</div>
-<div *ngIf="isLineDisplayed"
-  class="sky-progress-indicator-item-line"
-  [ngClass]="lineClassList"
->
-  <div *ngIf="isActive || isComplete"
-    class="sky-progress-indicator-item-content"
+  </sky-progress-indicator-status-marker>
+
+  <div
+    class="sky-progress-indicator-item-main"
```

Comment:
I used "main" to designate the main content of the step, and to insinuate the `main` HTML tag. I could use `sky-progress-indicator-item-body`, instead?

Reply:
hmm...`content` doesn't have a ton of semantic value, but we do have similar patterns elsewhere. I think I do like `content` better, if you don't mind?

Reply:
The word `main` in the CSS class isn't very descriptive... Could we find something a bit more semantic?

Reply:
Also, if this is simply for a flex container, don't we already have a sky class somewhere for that purpose?